cdap-cli/src/main/java/co/cask/cdap/cli/commandset/ScheduleCommands.java

```diff
@@ -0,0 +1,228 @@
+    public void perform(Arguments arguments, PrintStream printStream) throws Exception {
+      String[] programIdParts = arguments.get("app-id.schedule-id").split("\\.");
+      if (programIdParts.length < 2) {
+        throw new CommandInputError(this);
+      }
+
+      String appId = programIdParts[0];
+      String scheduleId = programIdParts[1];
+      scheduleClient.suspend(appId, scheduleId);
+      printStream.printf("Successfully suspended schedule '%s' in app '%s'", scheduleId, appId);
+    }
+
+    @Override
+    public String getPattern() {
+      return "suspend schedule <app-id.schedule-id>";
+    }
+
+    @Override
+    public String getDescription() {
+      return "Suspends a schedule";
+    }
+  }
+
+  /**
+   * Resumes a schedule.
+   */
+  public static final class ResumeScheduleCommand extends AbstractCommand {
+
+    private final ScheduleClient scheduleClient;
+
+    @Inject
+    public ResumeScheduleCommand(CLIConfig cliConfig, ScheduleClient scheduleClient) {
+      super(cliConfig);
+      this.scheduleClient = scheduleClient;
+    }
+
+    @Override
+    public void perform(Arguments arguments, PrintStream printStream) throws Exception {
+      String[] programIdParts = arguments.get("app-id.schedule-id").split("\\.");
+      if (programIdParts.length < 2) {
+        throw new CommandInputError(this);
+      }
+
+      String appId = programIdParts[0];
+      String scheduleId = programIdParts[1];
+      scheduleClient.resume(appId, scheduleId);
+      printStream.printf("Successfully resumed schedule '%s' in app '%s'", scheduleId, appId);
+    }
+
+    @Override
+    public String getPattern() {
+      return "resume schedule <app-id.schedule-id>";
+    }
+
+    @Override
+    public String getDescription() {
+      return "Resumes a schedule";
+    }
+  }
+
+  /**
+   * Lists the schedules for a given workflow.
+   */
+  private static final class ListWorkflowSchedulesCommand extends AbstractCommand {
+
+    private final ScheduleClient scheduleClient;
+    private final TableRenderer tableRenderer;
+
+    @Inject
+    public ListWorkflowSchedulesCommand(CLIConfig cliConfig, ScheduleClient scheduleClient,
+                                        TableRenderer tableRenderer) {
+      super(cliConfig);
+      this.scheduleClient = scheduleClient;
+      this.tableRenderer = tableRenderer;
+    }
+
+    @Override
+    public void perform(Arguments arguments, PrintStream printStream) throws Exception {
+      String[] programIdParts = arguments.get(ElementType.WORKFLOW.getArgumentName().toString()).split("\\.");
+      if (programIdParts.length < 2) {
+        throw new CommandInputError(this);
+      }
+
+      final String appId = programIdParts[0];
+      String scheduleId = programIdParts[1];
+      List<ScheduleSpecification> list = scheduleClient.list(appId, scheduleId);
+      Table table = Table.builder()
+        .setHeader("application", "program", "program type", "name", "description", "properties")
```

Comment:
I think you'd also want to print the type of the schedule, ie Time, or StreamSize - we only have those two as of now. You can figure it out by doing the following:

```
if (object.getSchedule() instanceof TimeSchedule) {
  // type is time
} else if (object.getSchedule() instanceof StreamSizeSchedule) {
  // type is stream size
}
```